src/scripts/main.js

```diff
@@ -22,7 +22,31 @@ const food = {
 const tree = document.querySelector('#tree');
 
 function createTree(element, data) {
-  // WRITE YOUR CODE HERE
+  tree.append(createTreeDom(data));
+}
+
+function createTreeDom(data) {
+  if (!Object.keys(data).length) {
```

Comment:
Лучше для читаемости
```suggestion
  if (Object.keys(data).length === 0) {
```